src/lib/airtable/patches/README.md

```diff
@@ -0,0 +1,10 @@
+### Overview
```

Comment:
this is so neat you are the best

Reply:
Should we also add preservations of non-generated functions in request.js that would be overwritten upon re-generation?

Reply:
Good idea. I'll do that in another PR that'll also remove the extra airtable field you included in the other comment